glide-core/src/client/value_conversion.rs

```diff
@@ -527,7 +527,7 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
 
     // TODO use enum to avoid mistakes
     match command.as_slice() {
-        b"HGETALL" | b"XREAD" | b"CONFIG GET" | b"FT.CONFIG GET" | b"HELLO" => {
+        b"HGETALL" | b"XREAD" | b"CONFIG GET" | b"FT.CONFIG GET" | b"HELLO" | b"XRANGE" => {
             Some(ExpectedReturnType::Map)
```

Comment:
small question, I see that the return type for this cmd is also a map of arrays of strings, why does an ExpectedReturnType of `Map` work here but a new ExpectedType had to be introduced for `LMPop`?

Reply:
I'm not sure why we didn't use `ExpectedReturnType::Map` for LMPOP actually. 
I believe it should return the same result... 
(with the one exception that LMPOP only works with exactly one key - but that's not really important)

Reply:
Confirmed.  
I changed LMPOP value conversion to `Some(ExpectedReturnType::Map)` and it works.  
@tjzhang-BQ you can consider cleaning that up unless you feel strongly that the array.len() == 1 check is important.